samples/DataImporter/GraphEngine.DataImporter/Importer.cs

```diff
@@ -261,11 +269,11 @@ public static IImporter GetImporter(string path)
                             }
```

Comment:
we should consider refactor this block into a separate method. for example "GetFileTypeByContent"